LICENSE

```diff
@@ -1,19 +0,0 @@
-Copyright (c) 2014 boreq
```

Comment:
You should actually include the full text of the GNU GPL if you wish to license your software under that license. I see you have included the preamble of the GNU GPL in this file, but in order to be fully protected under the GNU GPL you should consider including the full license text. I personally recommend the Affero GPL for this product :smile: